demisto_sdk/commands/update_release_notes/update_rn.py

```diff
@@ -169,7 +169,7 @@ def handle_existing_rn_version_path(self, rn_path: str) -> str:
                 The content of the rn
         """
         if self.existing_rn_version_path:
-            self.should_delete_existing_rn = self.existing_rn_version_path != rn_path
+            self.should_delete_existing_rn = os.path.abspath(self.existing_rn_version_path) != rn_path
```

Comment:
If you feel like improving this code, we can change and use a `Path` object instead (`Path(...).absolute() == ...`), and below we can `return path_object.read_text()`